solutions/silver/cf-1338A.mdx

```diff
@@ -65,6 +65,45 @@ int main() {
 ```
 
 </CPPSection>
+
+<JavaSection>
+```java
+import java.util.*;
+
+public class PoweredAddition {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		int testNum = sc.nextInt();
```

Comment:
i think 99% of java sols use bufferedreader

Reply:
ok